src/openclaw_pipeline/autopilot/daemon.py

```
@@ -718,7 +718,10 @@ def main():
     parser.add_argument(
         "--pack",
         default=None,
-        help="Domain pack 名称（默认: default-knowledge）",
+        help=(
+            f"Domain pack 名称（默认兼容 pack: {DEFAULT_PACK_NAME}；"
```

Ruff (0.15.9)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)